en/docs/services/create-your-first-service.md

```diff
@@ -18,70 +18,75 @@ To create the Choreo service, follow the procedure below:
 
 4. In the **Configure API Trigger** form, select **GET** as the HTTP method, and enter `activecases` as the relative path from host.
 
-    ![Resource Configuration](../assets/img/services/configure-api-trigger.png)
+    ![Resource Configuration](/en/assets/img/services/configure-api-trigger.png)
     
     Then click **Save API**. Now you have saved the API trigger that starts your application.
     
-5. In the next form that appears, do as follows to configure the HTTP connection that connects the public API used in this scenario to the API trigger you configured.
+5. Click **API Calls**, and then enter information as follows to configure the HTTP connection that connects the public API used in this scenario to the API trigger you configured.
 
-    1. Click **Connections**.
-
-    2. In the **New HTTP Connection** form, enter `https://api.covid19api.com` as the URL and select **GET** as the operation.
+    1. Click **HTTP**.
+    
+        ![Select HTTP Connection](/en/assets/img/services/select-http-connection.png)
+        
+    2. Enter information as follows:
     
-        ![New HTTP Connection](../assets/img/services/new-http-connection.png)
+        | **Field**           | **Value**                    |
+        |---------------------|------------------------------|
+        | **Connection Name** | `httpEndpoint`               |
+        | **URL**             | `https://api.covid19api.com` |
         
-        Then click **Save & Next**.
+        Click **Save**.
         
-    3. In the **Select Payload Type** field, select **JSON**.
+    3. Click the **+** icon below the HTTP API call you added. 
     
-        ![Select Payload Type](../assets/img/services/select-payload-type.png)
+        ![Update Existing Connection](/en/assets/img/services/update-existing-connection.png)
```

Comment:
Use relative paths. Check all the places. Changes were added in https://github.com/wso2/docs-choreo-dev/pull/63/files

Reply:
Corrected